docs/reference/Admin_Actions.md

```diff
@@ -203,3 +203,1432 @@ Modified some packages data on the backend database, due to incompatibility with
 - `ReeSilva/atom-homestead` - Removed `v0.10.0`
 - `kraih/atom-perltidy` - Removed `0.1`, `0.2`, `0.3`
 - `swdotcom/swdc-atom` - Removed Package
+
+## Pre-Backend
+
+In the time before the backend was actually created, when the Pulsar team had archived all existing packages from the Atom Package Registry, there were two steps taken that resulted in some packages not making it past the initial migration, and being excluded from the Pulsar Package Registry. Those steps are detailed below.
+
+### Database Migration
+
+During the initial migration of packages to the database, it's possible some packages would break some rules that had been configured within the schema of this new database. If you'd like the script and full logs of this process are available on [`confused-Techie/atom-package-migrator-to-db-2`](https://github.com/confused-Techie/atom-package-migrator-to-db-2/tree/main). Below are the package's that failed to migrate during this process:
+
+* `code-time`: Failed to migrate several versions due to the length of the license field being longer than what was allowed on the Database Schema. Since this was the only package to encounter this limit, we thought it best to contact the maintainers, rather than modify the database schema. An [issue](https://github.com/swdotcom/swdc-atom/issues/96) was created on `swdotcom/swdc-atom` to address this. But without response the package wasn't included as every version published encountered this issue.
+
+### Health Checks / Repackaging
+
+Before any archived package's were ever migrated, they were first put through several health checks, to ensure we only transfered over valid packages. The code used to run these health checks, and convert the archived data into usable package data is available on [`confused-Techie/atom-package-migrator`](https://github.com/confused-Techie/atom-package-migrator). The full log's and data created from this process is also available on [`confused-Techie/atom-package-collection`](https://github.com/confused-Techie/atom-package-collection/tree/main). Below will list all of the package's that failed these health checks.
+
+### Non-URL Safe Name
+
+Package's were excluded from the repackaging process if they contained a name that was not URL Safe.
+
+The below packages were never repackaged due to this:
+
+* ☃
+* itunes for atom
+* itunes for atom
+* panic-palette-syntax@0.2.5
+* undefined
+
+### Package's GitHub is not available
+
+Package's were excluded from the repackaging process if the repository they pointed to wasn't accessible. Indicating the repository or user has since been deleted, or the repository has since become privated.
```

Comment:
```suggestion
Package's were excluded from the repackaging process if the repository they pointed to wasn't accessible. This implies that the repository or user has since been deleted, or the repository has since become privated.
```